system/DetectNetworkConfigChangesSession.cpp

```diff
@@ -700,7 +700,7 @@ void DetectNetworkConfigChangesSession :: InternalThreadEntry()
   
    // This window will never be shown; its only purpose is to allow us to receive WM_POWERBROADCAST events so we can alert the calling code to sleep and wake events
    HWND hiddenWindow = CreateWindow(WINDOW_CLASS_NAME, _T(""), WS_OVERLAPPEDWINDOW, -1, -1, 0, 0, (HWND)NULL, (HMENU) NULL, NULL, (LPVOID)NULL); 
-   if (hiddenWindow) SetWindowLongPtr(hiddenWindow, GWL_USERDATA, (LONG) this);  // ok because we don't need access to (this) in WM_CREATE
+   if (hiddenWindow) SetWindowLongPtr(hiddenWindow, GWLP_USERDATA, (LONG) this);  // ok because we don't need access to (this) in WM_CREATE
```

Comment:
The "(LONG)" cast here is wrong, as the parameter type is LONG_PTR... The comment is also unnecessary.